ssl-cert.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Hello @BabdCatha, thanks for this PR! Can you please remove the certificates from this PR and add *.pem/*.key to .gitignore?